packages/ui-file-drop/src/FileDrop/index.tsx

```diff
@@ -136,7 +136,7 @@ class FileDrop extends Component<FileDropProps, FileDropState> {
     if (this.hasMessages) {
       return (
         this.props.messages!.findIndex((message: FormMessage) => {
-          return message.type === 'error'
+          return message.type === 'error' || message.type === 'newError'
```

Comment:
in FileDrop, the error icon isn't aligned with the text the same way as in the rest of the components (it seems to be aligned to the bottom end vs to the top in the rest)